modules/testing/pages/integration/canceling.adoc

```diff
@@ -0,0 +1,12 @@
+= Canceling Integration pipelineruns for Pull Requests
+
+Pushing a new change to your Pull Request creates a new build pipelinerun, followed by a new snapshot if build succeeded, then integration tests are triggered.
+In case you happen to push another change in your Pull Request, while integration tests pipelinerun are still running, the previous test pipelinerun will be canceled.
```

Comment:
```suggestion
In case you happen to push another change in your Pull Request while integration tests pipelineruns are still running, the previous test pipelineruns will be canceled.
```